lib/librato/rails/subscribers.rb

```diff
@@ -21,25 +21,12 @@ def self.watch_controller_action(controller, action)
             "#{controller}##{action}".freeze
           end
 
-        return @watches if @watches.include?(watch)
         @watches << watch
       end
 
-      def self.watch_controller_descendants_for(controller)
-        klass = controller.is_a?(String) ? controller.constantize : controller
-        return @watches if klass.descendants.empty? # base case
-
-        klass.descendants.each do |descendant|
-          Subscribers.watch_controller_action(descendant, :all)
-          Subscribers.watch_controller_descendants_for(descendant)
-        end
-
-        @watches
-      end
-
-      def self.track_controller_descendants
-        controllers = @watches.reject { |c| c.include?('#') } # specific controller actions do not have descendants
-        controllers.each { |c| Subscribers.watch_controller_descendants_for(c) }
+      def self.inherit_watches(base, descendant)
+        @watches ||= []
+        @watches << descendant.to_s.freeze if !!@watches.index(base.to_s)
```

Comment:
yeah, good point. I'd copied that from another place in the code where we were using `index` instead of `include?`